_posts/2024-04-15-trino-fest-2024-approaches.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "A sneak peek of Trino Fest 2024"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: true
+---
+
+Trino Fest is drawing ever closer, and Commander Bun Bun has been hard at work
+behind the scenes arranging the schedule and making sure that Trino's trip to
+Boston is going to be a great one. In case you missed it,
+[we announced Trino Fest]({{site.baseurl}}{% post_url 2024-02-20-announcing-trino-fest-2024 %})
+a couple months ago, and if you have missed it, make sure to go register to
```

Comment:
remove the duplication of "missed it" somehow

Reply:
That's on purpose. It's uh... anaphora. I'll add italics to make it more clear.